management/connection.go

```diff
@@ -704,6 +707,25 @@ type ConnectionOptionsSAMLIdpInitiated struct {
 	NonPersistentAttrs *[]string `json:"non_persistent_attrs,omitempty"`
 }
 
+type ConnectionOptionsGoogleApps struct {
+	ClientID     *string `json:"client_id,omitempty"`
+	ClientSecret *string `json:"client_secret,omitempty"`
+	Domain       *string `json:"domain,omitempty"`
+
+	EnableUsersAPI  *bool `json:"api_enable_users,omitempty"`
+	BasicProfile    *bool `json:"basic_profile,omitempty" scope:"basic_profile"`
+	ExtendedProfile *bool `json:"ext_profile,omitempty" scope:"ext_profile"`
+	Groups          *bool `json:"ext_groups,omitempty" scope:"ext_groups"`
+	Admin           *bool `json:"ext_admin,omitempty" scope:"ext_admin"`
+	IsSuspended     *bool `json:"ext_is_suspended,omitempty" scope:"ext_is_suspended"`
+	AgreedTerms     *bool `json:"ext_agreed_terms,omitempty" scope:"ext_agreed_terms"`
+
+	SetUserAttributes *string `json:"set_user_root_attributes,omitempty"`
+
```

Comment:
Could you also add `NonPersistentAttrs` as a value (so its consistent with the other connection objects), e.g. https://github.com/go-auth0/auth0/blob/master/management/connection.go#L253.